### ISSUE TITLE
Accuracy not improving on custom data and generated headline is entirely made up of UNK

### ISSUE THREAD

**Issue by SampannaKahu** [chen0040/keras-text-summarization#9]
Hi @chen0040 ,

Thanks for you code.

I am facing issue when running your model on custom data. The dataset that I am using is the CNN news dataset which contains news articles and their summary.

After training for 20 epochs for 1.2 GB of training data, I am only seeing an accuracy of 0.32

> 14361/14361 [==============================] - 5279s 368ms/step - loss: nan - acc: 0.3265 - val_loss: nan - val_acc: 0.3258
> Training done.

And one of the test results is:
Generated Headline:
> UNK UNK UNK UNK UNK UNK UNK UNK UNK UNK UNK UNK UNK UNK UNK UNK UNK UNK UNK UNK UNK UNK UNK UNK UNK UNK UNK UNK UNK UNK UNK UNK UNK UNK UNK UNK UNK UNK UNK UNK UNK UNK UNK UNK UNK UNK UNK UNK UNK UNK

Original Headline: 
> louis van gaal set to add key players to his manchester united side .. arjen robben , dani alves and kevin strootman on the wishlist .. diego godin and mats hummels are also linked with a move to old trafford .. manchester united have spent # 215m on players since september 2013 .. up to 10 first-team players could leave in january or in the summer .

And the weights file, even after training for 20 epochs on 1.2 GB training data, is 1.5 MB only.

Do you have any idea what the problem is?
